.github/workflows/windows_impl.yml

```diff
@@ -153,6 +151,4 @@ jobs:
           $env:MAMBA_TEST_SHELL_TYPE='powershell'
           Remove-Item -Path "env:CONDARC"
           # Only rerun flaky tests on the `main` branch
-          python -m pytest micromamba/tests/ \
-              ${{ runner.debug == 'true' && '-v --capture=tee-sys' || '--exitfirst' }} \
```

Comment:
As the lines are quiet long, we may think of using ` in windows as an equivalent to \ on linux.

Reply:
I agree.

I initially tried but I had an error as if `--exitfirst` was provided on a new line.